macros/context_existance_check.sql

```diff
@@ -0,0 +1,130 @@
+{#
+Copyright (c) 2023-present Snowplow Analytics Ltd. All rights reserved.
+This program is licensed to you under the Snowplow Personal and Academic License Version 1.0,
+and you may not use this file except in compliance with the Snowplow Personal and Academic License Version 1.0.
+You may obtain a copy of the Snowplow Personal and Academic License Version 1.0 at https://docs.snowplow.io/personal-and-academic-license-1.0/
+#}
+
+
+{% macro context_existance_check() %}
+  {% set contexts = [
+    ['snowplow__enable_mobile_context','contexts_com_snowplowanalytics_snowplow_mobile_context_1', 'snowplow__mobile_context'],
```

Comment:
The issue with this currently is that we tie the redshift/postgres context tables as vars I believe specifically to be able to overwrite them for integration tests which are running against the _stg tables, we need to allow for that to work. Can you try replacing them as vars?